src/src/java/DAO/EmployeeDAO.java

```diff
@@ -0,0 +1,105 @@
+package DAO;
+
+import Model.Employee;
+import Utils.PGConn;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author Metin
+ */
+public class EmployeeDAO {
+
+	private final Connection con = PGConn.getConnection();
+	private PreparedStatement ps;
+
+	public List<Employee> findAll() {
+		List<Employee> employees = new ArrayList<>();
+		try {
+			this.ps = this.con.prepareStatement("Select * from employee");
+			ResultSet rs = this.ps.executeQuery();
+			if (rs.next()) {
+				Employee tmp = new Employee(
+								rs.getInt("id"),
+								rs.getString("name"),
+								rs.getString("surname"),
+								rs.getString("phone"),
+								rs.getString("color"),
+								rs.getString("password"),
+								rs.getInt("roleId"));
+				employees.add(tmp);
+			}
+			return employees;
+		} catch (SQLException e) {
+			System.out.println("Error -->" + e.getMessage());
+		}
+		return employees;
```

Comment:
2 kere `return employees`  demek yerine `try {} catch {} finally {return employees;}` kullanilabilir sanirim bir dene.

Reply:
DAO'ları @aecr12 yazacak. EmployeeDAO'yu sadece login, logout ve session'ı test etmek için yazdım.